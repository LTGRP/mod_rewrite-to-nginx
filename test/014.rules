RewriteRule users/(.*)   /show?user=$1
