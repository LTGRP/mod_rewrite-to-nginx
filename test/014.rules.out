rewrite users/(.*) /show?user=$1;
